challenge_14/c/karanchawla/challenge_14.c

```diff
@@ -0,0 +1,90 @@
+//Karan Chawla
+//Challenge 14
+
+#include <stdio.h>
+#include <stdlib.h>
+
+//Linked list node definition
+typedef struct node
+{
+	char c;
+	struct node* next;
+}Node;
+
+
+//utility function to create a new node
+Node* newNode(char c)
+{
+	Node* NewNode = (Node*)malloc(sizeof(Node));
```

Comment:
Try to maintain consistency in your naming convention. Here you use camel case for a variable but below you use a more traditional lowercase with underscores.